package.json

```diff
@@ -25,5 +25,8 @@
     "stylelint": "^13.12.0",
     "stylelint-order": "^4.1.0",
     "stylelint-scss": "^3.19.0"
+  },
+  "devDependencies": {
+    "stylelint-config-standard-scss": "^2.0.1"
```

Comment:
Let's move it to the `dependencies` section

Reply:
I deleted devDependencies and changed  `stylelint-config-standard-scss` from "^23.0.0" to "^2.0.1"
 ```
"stylelint-config-standard-scss": {
      "version": "2.0.1",
      "resolved": "https://registry.npmjs.org/stylelint-config-standard-scss/-/stylelint-config-standard-scss-2.0.1.tgz",
      "integrity": "sha512-TW5NLquUSS0mg2N31zzaSbYRbV/CMifSVLdpgo6VdGvjysgYqJOcKM/5bmXucTOsdfqomcPXetFZ3adC7nD+cg==",
      "dev": true,
      "requires": {
       "stylelint-config-recommended-scss": "^5.0.0",
        "stylelint-config-standard": "^2.0.0"
      }
    },   
```